### PR TITLE
Update XDC integration test to use add remote cluster api

### PR DESCRIPTION
**Pull request by yux0**
**What changed?**
Update XDC integration test to use add remote cluster api

**Why?**
New release only support adding remote clusters via admin api.

**How did you test it?**
Integration tests

**Potential risks**

**Is hotfix candidate?**
No

### CODE REVIEW

File: common/cluster/metadata.go [hunk 1]
```diff
@@ -355,7 +365,7 @@ func (m *metadataImpl) refreshLoop(ctx context.Context) error {
 			for err := m.refreshClusterMetadata(ctx); err != nil; err = m.refreshClusterMetadata(ctx) {
 				m.logger.Error("Error refreshing remote cluster metadata", tag.Error(err))
 				select {
-				case <-time.After(refreshFailureInterval):
+				case <-time.After(m.refreshDuration() / 2):
```
Comment (yiminc) [inline common/cluster/metadata.go:368]: why /2?

File: host/testdata/xdc_integration_es_clusters.yaml [hunk 2]
```diff
@@ -38,14 +33,9 @@
   clustermetadata:
     enableGlobalNamespace: true
     failoverVersionIncrement: 10
-    masterClusterName: "active-es"
+    masterClusterName: "standby-es"
```
Comment (yiminc) [inline host/testdata/xdc_integration_es_clusters.yaml:36]: so both are master cluster?

File: common/cluster/metadata.go (hunk 1 above)
Comment (yux0) [reply common/cluster/metadata.go:368]: arbitrary number. I feel it just need one refresh internal config. No need to pass in a failure interval. But I am open to this.

File: host/testdata/xdc_integration_es_clusters.yaml (hunk 2 above)
Comment (yux0) [reply host/testdata/xdc_integration_es_clusters.yaml:36]: it doesn't matter here. I just try to mimic connecting two isolated clusters

**Review by yiminc — APPROVED** [review]

**Pull request merged**